R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -196,6 +196,12 @@ test_that("create DataFrame from RDD", {
   expect_equal(dtypes(df), list(c("name", "string"), c("age", "int"), c("height", "float")))
   expect_equal(as.list(collect(where(df, df$name == "John"))),
                list(name = "John", age = 19L, height = 176.5))
+  expect_equal(getNumPartitions(toRDD(df)), 1)
```

Comment:
Is the default always 1 or does it depend on the machine used ? I vaguely remember the default being inferred from number of cores available or something like that